src/Combinatorics/Graphs/functions.jl

```diff
@@ -1172,8 +1172,9 @@ end
 function graph_from_edges(::Type{T},
                           edges::Vector{Edge},
                           n_vertices::Int=-1) where {T <: Union{Directed, Undirected}}
-
-  n_needed = maximum(reduce(append!,[[src(e),dst(e)] for e in edges]))
+  isempty(edges) && return Graph{T}(n_vertices)
+  
+  n_needed = maximum(reduce(append!,[[src(e),dst(e)] for e in edges];))
```

Comment:
If I see it correctly, something like `graph_from_edges(Undirected, Edge[])` (with empty edge list and default third argument) won't work. This change should change that:
```suggestion
  n_needed = maximum(reduce(append!,[[src(e),dst(e)] for e in edges];); init=0)
```